docs/sources/clients/k6/_index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: k6 load testing
+weight: 90
+---
+
+# k6 Loki extension load testing
+
+Grafana [k6](https://k6.io) is a modern load-testing tool.
+Its clean and approachable scripting [API](https://k6.io/docs/javascript-api/)
+works locally or in the cloud.
+Its configuration makes it flexible.
+
+The [xk6-loki extension](https://github.com/grafana/xk6-loki) permits pushing logs to and querying logs from a Loki instance.
+It acts as a Loki client,
+simulating real-world load test scenarios on your Loki installation.
```

Comment:
In the intro, I miss the reasons you might want to load test your Lokin installation. Something along the line: 

> It acts as a Loki client, simulating real-world load to test the scalability, reliability, and performance of your Loki installation.